### PR TITLE
Update nuget package reference to latest available

### PR DESCRIPTION
**Pull request by davidhunter22**

### CODE REVIEW

File: src/IndexRange/IndexRange.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <VersionPrefix>1.0.0</VersionPrefix>
-    <TargetFrameworks>net45;net471;netstandard2.0;netstandard2.1</TargetFrameworks>
+    <TargetFrameworks>net45;net47;netstandard2.0;netstandard2.1</TargetFrameworks>
```
Comment (bgrainger) [inline src/IndexRange/IndexRange.csproj:5]: Why? Cf. bf40be9191567ede232229b1274ed9bc7a005ac3.

Comment (bgrainger) [reply src/IndexRange/IndexRange.csproj:5]: You're right; this was in .NET 4.7.

File: src/IndexRange/IndexRange.csproj [hunk 2]
```diff
@@ -30,19 +30,19 @@
...
-    <None Include="..\..\icon.png" Pack="true" PackagePath="\"/>
+    <None Include="..\..\icon.png" Pack="true" PackagePath="\" />
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'net45' ">
-    <PackageReference Include="System.ValueTuple" Version="4.5.0"/>
+    <PackageReference Include="System.ValueTuple" Version="4.5.0" />
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'net45' or '$(TargetFramework)' == 'net471' ">
```
Comment (bgrainger) [inline src/IndexRange/IndexRange.csproj:44]: This needs to be changed to `'net47'` so the reference assemblies are installed.

File: src/IndexRange/IndexRange.csproj (hunk 1 above)
Comment (davidhunter22) [reply src/IndexRange/IndexRange.csproj:5]: Ah I did downgrade 471 to 47 that was not my intent.
If anything it should be upgraded to 472

File: src/IndexRange/IndexRange.csproj (hunk 2 above)
Comment (davidhunter22) [reply src/IndexRange/IndexRange.csproj:44]: OK I can either switch it back to 471, make it 47 everywhere or make it 472 wverywhere, which do you prefer?

**Comment by bgrainger** [comment]
Fixed in https://github.com/bgrainger/IndexRange/commit/6292d638ddf4f0c5567b7c43b967485f48b3c836 and https://github.com/bgrainger/IndexRange/commit/0bbaae6b8f4c1d1a25031aee6e4dc464e88bddd9. Thanks for pointing this out!

File: src/IndexRange/IndexRange.csproj (hunk 1 above)
Comment (bgrainger) [reply src/IndexRange/IndexRange.csproj:5]: No, it should stay at `net47` to provide the greatest compatibility. (The advantage over `net45` is that it won't pull in an unnecessary `System.ValueTuple` dependency.)

**Pull request closed without merging**